sjrk.storyTelling.server.config.json5.example

```diff
@@ -6,16 +6,28 @@
             "server": {
                 "options": {
                     "globalConfig": {
-                        // the HTTP port the server will be hosted from, default is 8081
+                        /*
+                         * the HTTP port the server will be hosted from, default is 8081
+                         * this only needs to be set if you intend to use a different port
+                         */
                         "port": "8081",
 
-                        // the custom theme to load in on top of the base theme
+                        /*
+                         * the custom theme to load in on top of the base theme
+                         * for the base theme, simply remove, comment out or leave this setting empty
+                         * available themes are: karisma, learningReflections, cities, sojustrepairit
+                         */
                         "theme": "learningReflections",
 
-                        // a default file to load at the site root (e.g. index.html)
+                        /*
+                         * a default file to load at the site root (e.g. index.html)
+                         * if left blank or not set, this will default to storyBrowse.html
+                         */
                         "themeIndexFile": "introduction.html",
 
-                        // if true, saving stories to the database will be enabled
+                        /*
+                         * if true, saving stories to the database will be enabled
```

Comment:
It might be more accurate to say "adding and editing stories will be enabled".

Reply:
Good catch! I'll make that change

Reply:
what do you think of "creating and saving stories..." instead? "Editing" might imply that they can be changed after creation (I realize that the page is called "edit")

Reply:
Ya, the change makes sense.